content/en.json

```diff
@@ -174,18 +167,15 @@
     },
     "process-and-team": {
       "title": "Process and Team",
-      "subtitle": "Teams work <em>autonomously</em>, but usually pick an <em>agile</em> way of working. With a Product Owner from the customer.",
+      "subtitle": "Teams work <em>autonomously</em>, but usually pick an <em>agile</em> way of working with a Product Owner from the customer.",
       "how-title": "How we like to do this:",
       "how": [
-        "Team determines their own process",
-        "Continuously improve the process",
+        "Teams determine and improve their processes",
         "Product Owner from the client",
-        "Scrum master (can be a developer or project manager)",
+        "Scrum master",
         "Maintain a planning <em>together</em>",
-        "<em>All</em> team members help with refinement",
-        "Minimum of 2 people per <em>team</em>, split up from 6-7 upward",
-        "Call in help from colleagues in other teams",
-        "Regularly work together with the client at the same location",
```

Comment:
That line is differentiating for Infi I think, something I'd prefer to leave in there (at least in the PR that addresses "brevity", anything is up for discussion of course).

Reply:
Let's keep the differentiating stuff in there! (BTW I meant to remove ""Testers, designers, and other experts called in when needed", so I also did that now)